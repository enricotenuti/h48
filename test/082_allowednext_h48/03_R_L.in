0
2
R
L'

