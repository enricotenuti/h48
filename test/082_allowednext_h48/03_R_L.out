0x3F03F
